### ISSUE TITLE
Duplicate colors

### ISSUE THREAD

**Issue by bhaskarvk** [bpostlethwaite/colormap#9]
Certain Palettes generate duplicate values.
e.g. Temperature, jet etc.

**Comment by colinmorris** [comment]
Here's a small repro:

```javascript
var colormap = require('colormap');
var options = {
  colormap: 'jet',
  nshades: 12
};
var cm = colormap(options);
console.log(cm);
```

Outputs:

```js
[ '#000083',
  '#003caa',
  '#003caa',
  '#039ed5',
  '#05ffff',
  '#05ffff',
  '#82ff80',
  '#ffff00',
  '#ffff00',
  '#fd8000',
  '#fa0000',
  '#fa0000' ]
```

I've also observed this with the 'warm' palette. Increasing nshades doesn't seem to help.

**Comment by dfcreative** [comment]
@bhaskarvk it is going to be fixed in upcoming release. Stay tuned.

**Comment by bpostlethwaite** [comment]
solved in #15
